### ISSUE TITLE
Dataset version more important than registration date

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#3562]
**Dataset version more important than registration date**

Above the dataset title the date of registration with GBIF is shown very prominently. This is a rather useless date though for users. It would be far more informative to understand what version has been indexed. This could be the date of last successful indexing which should exist for all datasets consistently. But it could even be the pubDate of the source data given explicitly in the EML Anything is better than the date of registration.

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/19f73ac2c47faf27b6aab088000dbec8:d7fb44c4b50a7f5472940acb7944a921324a51e4f6b95f2c737e73122446916bb7be01ea5bab76a1883e17f2677857970b561efe120d5ad841e7c5ffdc5da6a2)
System: Safari 14.1.1 / Mac OS X 10.15.7
Referer: https://www.gbif.org/dataset/7ddf754f-d193-4cc9-b351-99906754a03b
Window size: width 1792 - height 1007
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2021-08-28T00:36:36.248Z',to:'2021-08-28T00:42:36.248Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2021-08-28T00:36:36.248Z',to:'2021-08-28T00:42:36.248Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by AbbyBenson** [comment]
Just chiming in to say that I think both are important. It can be helpful when I'm discussing the benefits of sharing data to GBIF and the data citations that come with that to note when the dataset was first registered so data providers understand how long it can sometimes take to start getting citations- e.g. a dataset registered in 2013 is going to have many more citations than a dataset registered in 2020. But if the page only shows the dataset version it may look like both are 2020 data.

**Comment by MortenHofft** [comment]
We have an area where we write other prominent information. Such as last modified and publication date. It would make sense so add version there as well. Has anything specifically prompted this @mdoering? Does it result in many questions to you? Or is it when looking at these pages yourself that this is a nuisance?

<img width="1210" alt="Screenshot 2021-08-31 at 09 20 45" src="https://user-images.githubusercontent.com/6759267/131460339-80e80ced-51a7-4d55-b33c-fd7e654cd88c.png">

**Comment by mdoering** [comment]
Ah, I was thinking we have "publication date" there, but it does not show for [COL](https://www.gbif.org/dataset/7ddf754f-d193-4cc9-b351-99906754a03b) so I filed this issue. That probably is even my bad when exporting the EML, I will take a look.

Still I think it would be better to show that pubDate at the very top instead of registration date. To me the registration is very technical and rather useless for regular users. But I guess it's there because it exists for all datasets and is very standardised?

The "version" is unfortunately not supported by EML. We could easily create of a new property in the additionalInformation extension though, I might actually place it there under COL already so it at least shows up in the EML. It is a very useful property for all datasets that have proper releases.

**Comment by mdoering** [comment]
Indeed we had a bug in the COL EML exporter that prevented pubDate to exist.

**Comment by MortenHofft** [comment]
Yes we show the registration date because it is always there (unlike publication date). I think it is a quality that it is consistently the same information (and not pubdate when available with a fallback to registration date). 

Honestly the information is mostly there for consistency with other pages that follow that form: `type | date`
`NEWS | 27 AUGUST 2021`, `OCCURRENCE | 30 JANUARY 2020` etc. Publication date would probably be better if it was always available. 

For missing dates and other key:value data we think is essential I think we should just make sure to add it in the above highlighted (red) area. And if it grows too long, then we can rethink the top "summary block".

I like the idea to add `version` to the API. I can create an issue in the registry project for it
